src/pages/app/Paginator.svelte

```diff
@@ -1,16 +1,10 @@
 <script>
-  import { search, initSearch, searchNext, searchPrev } from "@/search/search";
-  import { pushUrl } from "@/router/router";
-  import { queryBuilder } from "@/util/url";
-  import { simplePlural } from "@/util/string";
+  import { search, searchNext, searchPrev } from "@/search/search";
   import { _ } from "svelte-i18n";
 
   // SVG assets
   import leftPaginatorSvg from "@/assets/page_arrow_left.svg";
   import rightPaginatorSvg from "@/assets/page_arrow_right.svg";
-
-  export let dialog = false;
```

Comment:
Was dialog unused?

Reply:
It was. I was getting a warning about it, so I took it out.